webtau-groovy/src/main/groovy/org/testingisdocumenting/webtau/cfg/GroovyConfigBasedHttpConfiguration.groovy

```diff
@@ -49,7 +50,9 @@ class GroovyConfigBasedHttpConfiguration implements HttpConfiguration {
         }
 
         return HttpConfigurations.withDisabledConfigurations {
-            return headerProvider.get().call(fullUrl, passedUrl, given) as HttpHeader
+            HttpValidationHandlers.withDisabledHandlers {
```

Comment:
Now I wonder if we are going to have the same issue in Java. Could you please add java full example in a separate PR when you have time?

Reply:
I created an equivalent Java test and we definitely have the same problem :(

Reply:
I'll do that in a separate PR.